instances.go

```diff
@@ -16,22 +16,43 @@ package kp
 
 import (
 	"context"
+	"encoding/json"
+	"errors"
 	"fmt"
 	"net/url"
 	"time"
 )
 
+type PolicyType int
+
 const (
 	//DualAuthDelete defines the policy type as dual auth delete
-	DualAuthDelete = "dualAuthDelete"
+	DualAuthDelete PolicyType = iota
```

Comment:
Why was this changed from strings to ints? There is a lot of casting to String() in this code. I feel that a more natural way would be leaving the consts as strings.

Then remove the type `PolicyType`.

Right after the `const` you could declare an array of strings to use in the `Contains()` function.

`var PolicyTypes = [...]string{AllowedIP, AllowedNetwork, DualAuthDelete, Metrics}`

Reply:
This is an enum approach and we have used similar approach in KP code. 
However after discussion with Matt O, decided to revert it to strings in order not to disturb the global constants.